packages/aws-cdk-lib/aws-cloudfront/lib/realtime-log-config.ts

```diff
@@ -0,0 +1,96 @@
+import { Construct } from 'constructs';
+import { CfnRealtimeLogConfig } from './cloudfront.generated';
+import { IResource, Resource } from '../../core';
+
+/**
+ * Endpoint data streams types.
+ */
+export enum DataStreamType {
+  KINESIS = 'Kinesis'
+}
+
+/**
+ * Contains information about the Amazon Kinesis data stream where you are sending real-time log data.
+ */
+export interface KinesisStreamConfig {
+  /**
+   * The Amazon Resource Name (ARN) of an AWS Identity and Access Management (IAM) role that CloudFront can use to send real-time log data to your Kinesis data stream.
+   */
+  readonly roleArn: string;
+  /**
+   * The Amazon Resource Name (ARN) of the Kinesis data stream where you are sending real-time log data.
+   */
+  readonly streamArn: string;
+}
+
+/**
+ * Contains information about the Amazon Kinesis data stream where you are sending real-time log data in a real-time log configuration.
+ */
+export interface EndPoint {
+  /**
+   * Contains information about the Amazon Kinesis data stream where you are sending real-time log data.
+   */
+  readonly kinesisStreamConfig: KinesisStreamConfig,
+  /**
+   * The type of data stream where you are sending real-time log data.
+   */
+  readonly streamType: DataStreamType;
+}
+
+/**
+ * Represents Realtime Log Configuration
+ */
+export interface IRealtimeLogConfig extends IResource {
+  /**
+   * The name of the realtime log config.
+   */
+  readonly realtimeLogConfigName: string;
+  /**
+   * The arn of the realtime log config.
+   */
+  readonly realtimeLogConfigArn: string;
```

Comment:
Since these are attributes, we should also be using the `@attribute` tag. So,
```ts
**
* The name of the realtime log config.
*
* @attribute
*/
readonly realtimeLogConfigName: string;
```

Reply:
that makes sense